client/src/node/extension.ts

```diff
@@ -55,8 +55,11 @@ const activeProfileStatusBarIcon = window.createStatusBarItem(
   0,
 );
 
+export let extensionContext: ExtensionContext | undefined;
```

Comment:
There's a `client/src/components/ExtensionContext` trying to encapsulate it. How about providing get/setSecret there?

Reply:
Ha. Didn't know about this one. Updated in https://github.com/sassoftware/vscode-sas-extension/pull/592/commits/551093751ab18c3476a50ad45388a9da33c54df9